src/shared/components/common/markdown-textarea.tsx

```diff
@@ -144,10 +144,10 @@ export class MarkdownTextArea extends Component<
             !this.state.submitted
           }
         />
-        <div className="form-group row">
+        <div className="input-group mb-3 row">
```

Comment:
`input-group` has some side effects.

With `input-group`:

<img width="1151" alt="Screenshot 2023-06-19 at 5 50 24 PM" src="https://github.com/LemmyNet/lemmy-ui/assets/35377827/e672d7b3-5829-4f7b-b5a4-1582c9155621">

Without:

<img width="1141" alt="Screenshot 2023-06-19 at 5 50 15 PM" src="https://github.com/LemmyNet/lemmy-ui/assets/35377827/5c7d97d8-1c52-42d0-89a6-d96fe5faea0e">

Note how this `MarkdownTextarea` doesn't scale to the full width of the column.

Reply:
I also seriously wish we could get all those buttons on one line. Maybe we could put the help icon next to the label instead? 🤔

Reply:
If we could fit this all on one row, it would be _ perfection _.

Reply:
Ah, I think this is because I have some instances of `input-group` and `row` classes on the same element; it seems not to like that.

Reply:
> If we could fit all these icons all on one row, and have the textarea properly fit the space, it would be _ perfection 

Also remember, even if it fit, it would barely fit, so for anybody who increased their browser zoom or were on mobile, it would still wrap. It's a tight fit. I think it wraps kind of nicely, it doesn't bother me too too much.